### PR TITLE
Vet Clinic Database #2: Update table

### PR DESCRIPTION
**Pull request by miltonHenschel**
- [x] Use database transactions.

- [x] Modify and delete data in SQL.

- [x] Prepare complex queries that answer analytical questions.

- [x] Take a screenshot of the results.

- https://github.com/miltonHenschel/vet-clinic-db/blob/2-update-table/screenshots/2-query/actions1.jpg
- https://github.com/miltonHenschel/vet-clinic-db/blob/2-update-table/screenshots/2-query/actions2.jpg
- https://github.com/miltonHenschel/vet-clinic-db/blob/2-update-table/screenshots/2-query/actions3.jpg
- https://github.com/miltonHenschel/vet-clinic-db/blob/2-update-table/screenshots/2-query/query1.jpg
- https://github.com/miltonHenschel/vet-clinic-db/blob/2-update-table/screenshots/2-query/query2.jpg
- https://github.com/miltonHenschel/vet-clinic-db/blob/2-update-table/screenshots/2-query/query3.jpg
- https://github.com/miltonHenschel/vet-clinic-db/blob/2-update-table/screenshots/2-query/query4.jpg
- https://github.com/miltonHenschel/vet-clinic-db/blob/2-update-table/screenshots/2-query/query5.jpg
- https://github.com/miltonHenschel/vet-clinic-db/blob/2-update-table/screenshots/2-query/query6.jpg

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Henschel, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Kindly ensure that all main headers from the [ReadMe template](https://github.com/microverseinc/readme-template) should be present in your ReadMe, only the _FAQ_ section is optional.
-  Please add the _Key Features_ section of your ReadMe and include 1-3 key features of the project. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: queries.sql [hunk 1]
```diff
@@ -23,3 +23,30 @@ WHERE name != 'Gabumon';
...
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE EXTRACT(year FROM date_of_birth) >= 2022;
+SAVEPOINT delete;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO delete;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
```
Comment (Lucas-Masaba) [inline queries.sql:52]: - [ ] Please add the SQL code for the queries here as mentioned in the project requirements.
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @miltonHenschel  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Descriptive pull request :scroll: 
- Made changes to README.md :surfer: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (miltonHenschel) [reply queries.sql:52]: Thanks @Lucas-Masaba. I forgot to include them. You can see that I already included the screenshots of the resulting queries. Lemme do so ASAP...

**Review by afanimartin — APPROVED** [review]
Hi @miltonHenschel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Changes made as requested

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

## Find me on slack
### Martin Afani

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — COMMENTED** [review]
Hi Henschel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job adding the species
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**